tests/components/slots.test.ts

```diff
@@ -74,6 +74,24 @@ describe("slots", () => {
     expect(fixture.innerHTML).toBe("<span>other text</span>");
   });
 
+  test("simple named and empty slot", async () => {
+    class Child extends Component {
+      static template = xml`<span><t t-slot="default" /><t t-slot="myEmptySlot"/></span>`;
```

Comment:
can you add default-content to `myEmptySlot` to check that it gets overwritten?

Reply:
done, and nice catch :) 
the resulting code and templates are even better